gcpkms:// parsing is broken in 1.4.x

## Symptom

When trying to use cosign with GCP KMS, the `--key gcpkms://` is not parsed properly and handled as a local file path.

Affected versions:

- `v1.4.0`
- `v1.4.1`

Example run error below:

```bash
$ cosign public-key --key "gcpkms://projects/xxx/locations/global/keyRings/container-images/cryptoKeys/xxx/versions/1"
Error: open gcpkms:/projects/xxx/locations/global/keyRings/container-images/cryptoKeys/xxx/versions/1: no such file or directory
main.go:46: error during command execution: open gcpkms:/projects/xxx/locations/global/keyRings/container-images/cryptoKeys/xxx/versions/1: no such file or directory
```

It works well with `v1.3.x`

--- a/pkg/signature/keys.go
+++ b/pkg/signature/keys.go
@@ -18,6 +18,7 @@ import (
 	"context"
 	"crypto"
 	"crypto/x509"
+	"fmt"
 	"os"
 	"path/filepath"
 	"strings"
@@ -140,12 +141,14 @@ func SignerVerifierFromKeyRef(ctx context.Context, keyRef string, pf cosign.Pass
 		}
 
 		return cosign.LoadECDSAPrivateKey([]byte(pk), []byte(pass))
-	default:
-		if sv, err := kms.Get(ctx, keyRef, crypto.SHA256); err == nil {
-			return sv, nil
-		}
 	}
 
+	sv, err := kms.Get(ctx, keyRef, crypto.SHA256)
+	if err == nil {
+		return sv, nil
+	}
+
+	fmt.Fprintf(os.Stderr, "an error occurred: %v, will try to load key from disk...\n", err)
 	return loadKey(keyRef, pf)
 }
 

